src/closestPair.js

```diff
@@ -0,0 +1,26 @@
+export default function closestPair(points) {
+
+  let lowestDistance = pythagorean(points[0], points[1])
```

Comment:
you're giving `lowestDistance` a value here but then (i think) you always override in the for loops later. At the least I think you're executing `pythagorean(points[0], points[1])` more than once. You should avoid doing calculations twice.